src/server/pfs/server/server_test.go

```diff
@@ -680,7 +680,7 @@ func TestAncestrySyntax(t *testing.T) {
 
 	commit1, err := client.StartCommit(repo, "master")
 	require.NoError(t, err)
-	_, err = client.PutFileOverwrite(repo, commit1.ID, "file", strings.NewReader("1"), 0)
+	_, err = client.PutFileOverwrite(repo, "master", "file", strings.NewReader("1"), 0)
```

Comment:
Nothing changed, I rewrote this to not explicitly start and finish commits in which case you don't have `commit1` in scope, but then changed it back because I need it a little bit farther down. Either way should have the same semantics.